modules/Bio/EnsEMBL/Compara/PipeConfig/Vertebrates/StrainsOrthologQM_Alignment_conf.pm

```diff
@@ -0,0 +1,56 @@
+=head1 LICENSE
+
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2019] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+=head1 NAME
+
+Bio::EnsEMBL::Compara::PipeConfig::Vertebrates::StrainsOrthologQM_Alignment_conf
+
+=head1 SYNOPSIS
+
+    init_pipeline.pl Bio::EnsEMBL::Compara::PipeConfig::Vertebrates::StrainsOrthologQM_Alignment_conf -host mysql-ens-compara-prod-X -port XXXX \
+        -member_type <protein_or_ncrna> -collection <murinae_or_sus>
+
+=head1 DESCRIPTION
+
+This pipeline uses whole genome alignments to calculate the coverage of
+homologous pairs for Vertebrates strains. See parent class for additional
+information.
+
+=cut
+
+package Bio::EnsEMBL::Compara::PipeConfig::Vertebrates::StrainsOrthologQM_Alignment_conf;
+
+use strict;
+use warnings;
+
+use base ('Bio::EnsEMBL::Compara::PipeConfig::Vertebrates::OrthologQM_Alignment_conf');
+
+
+sub default_options {
+    my ($self) = @_;
+    return {
+        %{$self->SUPER::default_options},   # inherit the generic ones
+
+        'collection'  => undef, # undef to force to be passed as an argument
```

Comment:
This doesn't force anything. There will be Perl warnings such as
> Use of uninitialized value in substitution iterator at /home/matthieu/workspace/src/hive/master/modules/Bio/EnsEMBL/Hive/DependentOptions.pm line 200.
and the database will be named like `matthieu_vertebrates__protein_orth_qm_wga_99`

To really force something you can:
- either edit `$self->SUPER::default_options` to remove `collection`, but we never do this pattern
- or introduce an alias like `'collection' => $self->o('collection_name'),`